.github/workflows/build.yml

```diff
@@ -28,8 +28,10 @@ jobs:
             shorthand: etwh
           - name: heatpump_water_heater
             shorthand: hpwh
-          - name: hvac
-            shorthand: hvac
```

Comment:
I wonder if we should leave this for now. Presumably it will just generate a (largely empty) HVAC build. But at least that will populate to the Web Tools while we figure out the Air Handler/Furnace change there?